src/gui/mzroll/point.cpp

```diff
@@ -144,6 +144,11 @@ void EicPoint::mousePressEvent (QGraphicsSceneMouseEvent* event) {
 
     if ( _group && _group->isIsotope() == false ) {
         _mw->isotopeWidget->updateIsotopicBarplot(_group);
+        _mw->isotopeWidget->setPeakGroupAndMore(_group);
```

Comment:
@shubhra-agrawal Can you document this part? Why this else condition?

Reply:
@sahil21 done.